pkg/virt-controller/watch/drain/evacuation/evacuation.go

```diff
@@ -403,8 +403,10 @@ func (c *EvacuationController) sync(node *k8sv1.Node, vmisOnNode []*virtv1.Virtu
 		return nil
 	}
 
-	diff := int(math.Min(float64(freeSpots), float64(len(migrationCandidates))))
-	remaining := freeSpots - diff
+	vmsToEvacuate := int(math.Min(float64(freeSpots), float64(len(migrationCandidates))))
```

Comment:
How can this happen?
`if freeSpots <= 0 {
		c.Queue.AddAfter(node.Name, 5*time.Second)
		return nil
	}`
leaving only `migrationCandidates` to be the suspect. But that doesn't seem to be able to be `<0`.

I think the version of Kubevirt in the bug does not include https://github.com/kubevirt/kubevirt/commit/5d1b049a5154c72e1b888da5ca392a9b97858995

Reply:
Yup, you're right! Thanks!

Do you think it makes sense to keep this as another protection layer?
I mean, if we have another bug in this area, we at least won't panic.

WDYT?

Reply:
I think for this case there are already unit tests present. I am definitely for improving coverage if anything. It is hard to argue about the need for this additional line as someone as well can argue it is redundant and therefore propose removal of it.

Reply:
Yeah, makes sense. Alright, if it's already taken care of with unit tests I think we can close this for now.
/close